joomla/joomla-cms
# [4.0] Fix language strings causing false plurals in crowdin

Pull Request for https://github.com/infograf768/j4localise/issues/30#issuecomment-925585512 .

### Summary of Changes

Language string constants ending with "_ONE" seem to trigger some kind of plural string generation for translations on Crodwin.

See https://github.com/infograf768/j4localise/issues/30#issuecomment-925585512 .

To avoid this, we should not use language string constants ending with "_ONE" if they are not really plural strings.

Work in Progress (WiP): I have to complete testing instructions.

As soon as ready, I'll remove draft status from this PR.

### Testing Instructions

Hint: It needs to run "npm ci" or "npm run build:js" after having applied the PR.

1. Code review: Check if there are language strings in the CMS core having a key ending with "_ONE".
2. In content history modal of some content having some history (i.e. more than 1 version), use the "Preview" button without having selected any version.
Hint: You can use that button even if it visually appears to be disabled. But that's another issue to be solved with another PR.
3. In the same modal, use the "Compare" button with only one version selected.
4. Create a module of type "Tags - Similar" and check the available options in the "Match Type" drop-down on the first Tab "Module".

### Actual result BEFORE applying this Pull Request

1. Strings "COM_CONTENTHISTORY_BUTTON_SELECT_ONE" and "MOD_TAGS_SIMILAR_FIELD_ONE" found.

2.
<img width="1042" alt="Screen Shot 2021-09-24 at 07 18 06" src="https://user-images.githubusercontent.com/7413183/135750404-0b7aad5f-588a-48bb-932c-5a8be4dc1290.png">

3.
<img width="1037" alt="Screen Shot 2021-09-24 at 07 18 24" src="https://user-images.githubusercontent.com/7413183/135750420-59bb3826-fc09-4b71-ae01-fae5c2eb6295.png">

4. Options "All", "Any" and "Half" are available.

### Expected result AFTER applying this Pull Request

1. None.

2. Same as without this PR.

3. Same as without this PR.

4. Same as without this PR.

### Documentation Changes Required

None.
